make.cmd

```diff
@@ -0,0 +1,59 @@
+@echo off
+
+setlocal
+
+set GOBIN=main
+set OUT=function.zip
+
+if "%1"=="" goto :BUILD_ALL
+if "%1"=="all" goto :BUILD_ALL
+if "%1"=="godev" goto :GO_DEV
+if "%1"=="clean" goto :CLEAN
+
+echo Invalid make target "%1"
+exit /b 2
+
+:BUILD_ALL
+    call :GCF_JS
+    if %errorlevel% neq 0 exit /b %errorlevel%
+
+    call :GCF_GO
+    if %errorlevel% neq 0 exit /b %errorlevel%
+
+    powershell -command "& {& 'Compress-Archive' -DestinationPath %OUT% -Path %GOBIN%,node_modules,index.js,package.json -Force}"
```

Comment:
If you are going to depend on PowerShell, maybe it would be better to make this whole file a PowerShell script?

Reply:
I decided to use a batch script instead because the most recent windows update, command prompt was the default terminal and can be run just by calling the name of the file.

Additionally, running PowerShell scripts is disabled by default.